CONSTITUTION.md

```diff
@@ -55,11 +55,19 @@ Membership shall be open to all Union members, and to all other persons satisfyi
 
 - 4.2.5 Annual subscription is dated for the calendar year, expiring on the 31st of December of that year.
 
-### 4.3 Disqualification
+### 4.3 Expulsion
 
-- 4.3.1 A person being one month in arrears in annual subscription shall cease to be a member.
```

Comment:
As discussed on Discord, we probably should retain this clause, since it's always a good idea to have members at all times. At the very least, the Exec should renew their memberships at the beginning of the year in order to maintain continuity.

If we want to go down the path of removing this clause, this really should be a separate motion.